crates/miden-lib/src/account/faucets/mod.rs

```diff
@@ -220,6 +223,32 @@ impl TryFrom<&Account> for BasicFungibleFaucet {
 // FUNGIBLE FAUCET
 // ================================================================================================
 
+/// Extension trait for fungible faucet accounts. Provides methods to access the fungible faucet
+/// account's reserved storage slot.
+pub trait FungibleFaucetExt {
+    const ISSUANCE_ELEMENT_INDEX: usize;
+    const ISSUANCE_STORAGE_SLOT: u8;
+
+    /// Returns the issuance of the fungible faucet account.
```

Comment:
I would write this as: "Returns the amount of tokens (in base units) issued from this fungible faucet."